### PR TITLE
updates package cloud repo secure key

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

**Comment by onurctirtir** [comment]
It seems that we should also update github token:

>$ build_new_release
Your token was rejected by GitHub.

**Comment by onurctirtir** [comment]
Seems that we need to fix year in this line https://github.com/citusdata/packaging/blob/debian-cron-packagecloud-keyfix/debian/changelog#L13

**Comment by onurctirtir** [comment]
Also, @gurkanindibay, you need to sign cla, it's a one-time thing

>❌ gurkanindibay sign now
You have signed the CLA already but the status is still pending? Let us recheck it.

**Comment by onurctirtir** [comment]
It's pretty normal that ci fails after adding ubuntu/focal to build matrix, to fix this, we need to use latest version of tools repo here:
https://github.com/citusdata/packaging/blob/debian-cron-packagecloud-keyfix/.travis.yml#L22

https://github.com/citusdata/tools/releases/tag/v0.7.20

**Comment by gurkanindibay** [comment]
> Also, @gurkanindibay, you need to sign cla, it's a one-time thing
> 
> > ❌ gurkanindibay sign now
> > You have signed the CLA already but the status is still pending? Let us recheck it.

Could you define me how to sign cla?

File: .travis.yml [hunk 1]
```diff
@@ -17,8 +17,9 @@ env:
     - TARGET_PLATFORM=debian/wheezy
     - TARGET_PLATFORM=ubuntu/xenial
     - TARGET_PLATFORM=ubuntu/trusty
+    - TARGET_PLATFORM=ubuntu/focal
 before_install:
-  - git clone -b v0.7.1 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b v0.7.2 --depth 1 https://github.com/citusdata/tools.git
```
Comment (onurctirtir) [inline .travis.yml]: should be 0.7.20, not 0.7.2, that's why ci is failing on ubuntu/focal

**Comment by onurctirtir** [comment]
we should remove this line, wheezy reached to eol
https://github.com/citusdata/packaging/blob/debian-cron-packagecloud-keyfix/.travis.yml#L17

File: .travis.yml [hunk 2]
```diff
@@ -43,5 +43,5 @@ deploy:
     skip_cleanup: true
     local-dir: "pkgs/nightlies"
     on:
-      branch: "debian-cron"
+      branch: "debian-cron-packagecloud-keyfix"
```
Comment (onurctirtir) [inline .travis.yml]: changing this line is a lot dangerous, please revert this commit before merging

File: .travis.yml [hunk 3]
```diff
@@ -19,7 +19,7 @@ env:
     - TARGET_PLATFORM=ubuntu/trusty
     - TARGET_PLATFORM=ubuntu/focal
 before_install:
-  - git clone -b v0.7.20 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b v0.7.20_test --depth 1 https://github.com/citusdata/tools.git
```
Comment (onurctirtir) [inline .travis.yml]: I think we can also revert this change

**Comment by onurctirtir** [comment]
To describe what this pr does for further readers, I would write something like below to pr description:

```
* Updates packagecloud & github secure keys
* Fixes a build issue due to invalid date in changelog
* Removes debian/wheezy support
* Removes build dependencies that don't exist anymore

Finally,
* Adds ubuntu/focal support
* Also, to support ubuntu/focal, updates tools version to 0.7.20
```

Also, might be good to modify pr title to something like:
> Fixes several issues & adds ubuntu/focal support

------

Also, I don't know which one is better but in general, we start commit messages with capital letter and we use verbs as is, like in Jelte's commit message:
>**R**emove ...
Fix x ..
Update x to solve y

You don't need to modify your commit messages in this pr but I wanted to write those for your further pull requests.
